HorizonApache/src/main/java/com/hubspot/horizon/apache/internal/ProxiedSSLSocketFactory.java

```diff
@@ -32,7 +33,13 @@ public static SSLConnectionSocketFactory forConfig(SSLConfig config) {
   }
 
   private static SSLConnectionSocketFactory acceptAllSSLSocketFactory() throws GeneralSecurityException {
-    SSLContext sslContext = SSLContexts.custom().loadTrustMaterial(null, new TrustAllTrustStrategy()).build();
+    SSLContext sslContext = SSLContexts.custom().loadTrustMaterial(null, new TrustStrategy() {
+
+      @Override
+      public boolean isTrusted(X509Certificate[] x509Certificates, String s) throws CertificateException {
```

Comment:
I think you can do a lambda here, eg `(certChain, authType) -> true`

Reply:
Very cool! I went ahead and updated the PR with the lambda!